sdk/recoveryservices-siterecovery/Azure.ResourceManager.RecoveryServicesSiteRecovery/src/Generated/Models/VMwareCbtEnableMigrationInput.cs

```diff
@@ -79,6 +79,10 @@ public VMwareCbtEnableMigrationInput(string vmwareMachineId, IEnumerable<VMwareC
         public string TargetAvailabilityZone { get; set; }
         /// <summary> The target proximity placement group ARM Id. </summary>
         public string TargetProximityPlacementGroupId { get; set; }
+        /// <summary> The confidential VM key vault Id for ADE installation. </summary>
+        public string ConfidentialVmKeyVaultId { get; set; }
```

Comment:
What's the format of this property value?
- If it's a GUID, use type [Guid](https://learn.microsoft.com/en-us/dotnet/api/system.guid?view=net-6.0). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#d4cebe383870410cab2e14d0709f78bb).
- If it's a resource identifier, use type [ResourceIdentifier](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/ResourceIdentifier.cs). See example [here](https://apiview.dev/Assemblies/Review/25a98e223bda4f39b4ef76bebffa2c23#9f6fc0ae7c8e497e98d7979dadf97a4c).

Reply:
`ConfidentialVmKeyVaultId` is now `ResourceIdentifier`.